etc/ncp-config.d/namecheapDNS.cfg

```
@@ -0,0 +1,46 @@
+{
+  "id": "namecheapDNS",
+  "name": "Dynamic DNS from Namecheap",
+  "title": "namecheapDNS",
+  "description": "Namecheap Dynamic DNS service (need domain from https://www.namecheap.com/)",
+  "info": "First, complete these steps: https://www.namecheap.com/support/knowledgebase/article.aspx/595/11/how-do-i-enable-dynamic-dns-for-a-domain/ and https://www.namecheap.com/support/knowledgebase/article.aspx/43/11/how-do-i-set-up-a-host-for-dynamic-dns",
+  "infotitle": "",
+  "params": [
+    {
+      "id": "ACTIVE",
+      "name": "Active",
+      "value": "no",
+      "type": "bool"
+    },
+    {
+      "id": "PASSWORD",
+      "name": "Dynamic DNS password",
+      "value": "",
+      "suggest": "840deed9ef1844eca7c198df2d31bf10"
```

```suggestion
      "suggest": "your_password"
```

also this should be "type: password"